__tests__/shutdown.test.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+/* eslint-env jest */
```

Comment:
We can configure this in `.eslintrc` instead of on a per file basis! I'll make a PR for that.